casa/OS/HostInfo.cc

```diff
@@ -144,11 +146,15 @@ ptrdiff_t HostInfo::memoryTotal(bool use_aipsrc) 		\
     /** returns the memory in kilobytes...         **/		\
     if ( use_aipsrc ) {						\
 	String value;						\
-	if ( Aipsrc::find(value, memory) ) {			\
+    if (resources_memory > 0) {				\
+        return resources_memory * 1024;			\
```

Comment:
the conversion to kb should be done in the setter and not here, but more importantly it is inconsistent again. The getters return kb, but the setter wants mb.
yes the aipsrc is different but as before consistency of the class is more important.

(btw. it might be a good idea to add units to the aipsrc to make this unambiguous)